ui/src/frame.tsx

```diff
@@ -32,7 +32,12 @@ const
     },
     body: {
       flexGrow: 1,
+      position: 'relative',
     },
+    frame: {
+      position: 'absolute',
```

Comment:
It would be good to add your PR comment here, so that we don't inadvertently revert the change in the future.

Reply:
Done.